core/templates/pages/topic-editor-page/navbar/topic-editor-navbar.directive.ts

```diff
@@ -146,6 +146,12 @@ angular.module('oppia').directive('topicEditorNavbar', [
           };
 
           $scope.isTopicSaveable = function() {
+            if ($scope.topicRights.isPublished()) {
+              return (
+                $scope.getChangeListLength() > 0 &&
+                $scope.getWarningsCount() === 0 &&
+                $scope.prepublishValidationIssues.length === 0);
```

Comment:
Should we simplify this to (and avoid duplicate lines):
```
return (
                $scope.getChangeListLength() > 0 &&
                $scope.getWarningsCount() === 0 && (
                !$scope.topicRights.isPublished() || $scope.prepublishValidationIssues.length === 0));

```

Reply:
Done, thanks!